python/paddle/nn/layer/loss.py

```diff
@@ -13,9 +13,93 @@
 # limitations under the License.
 
 # TODO: define loss functions of neural network  
-# __all__ = ['NCELoss',
-#            'CrossEntropyLoss',
-#            'MSELoss',
-#            'L1Loss',
-#            'NLLLoss',
-#            'BCELoss']
+import paddle.fluid as fluid
+__all__ = [
+    #'NCELoss',
+    #    'CrossEntropyLoss',
+    #    'MSELoss',
+    'L1Loss',
+    #    'NLLLoss',
+    #   'BCELoss'
+]
+
+
+class L1Loss(fluid.dygraph.Layer):
+    """
+    This interface is used to construct a callable object of the ``L1Loss`` class.
+    The L1Loss layer calculates the mean absolute error of input predictions and target label.
```

Comment:
Not only `mean absolute error`? `reduction` may be `sum`, `mean` or `none`.

Reply:
Done, thanks.